content/chainguard/chainguard-images/images-directory/index.md

```diff
@@ -0,0 +1,133 @@
+* **URI**: the registry URI you can use in a `docker pull` command to download the Image. Production Images, which are not publicly available, will instead have a message reading "Contact us for access to this image" with a link to Chainguard's [contact form](https://www.chainguard.dev/contact?utm_source=docs).
+* **Available versions**: a list of what versions are available for this Image. Each Image will have a `latest` version, and most will also have a `latest-dev` version. 
+* **Updated**: how long it's been since the Image was last updated. 
+
+You can click on any of these column names to sort the list of Images in ascending or descending order based on the values in these columns.
+
+Above the table is a search box you can use to find specific Images by their name or latest version number. To the right of this box is a drop-down menu labeled **Category**. You can use this to filter the Images listed based on what category they belong to.
+
+
+## Image information
+
+Next, let's inspect an individual Image. Click on any Image you'd like. This example shows the page for the `postgres` Image.
+
+![Screenshot of the Image Details page for the postgres Image, showing the "Versions" tab.](imgs-dir-4.png)
+
+Each Image page has five tabs that provide information about various facets of the given Image.
+
+
+### Versions Tab
+
+The default page for each Image is the **Versions** tab which contains information about the versions available for each Image. This contains a table with columns:
+
+* **Version**: this column lists each version tag available for the Image.
+* **Pull URL**: The URL you can use to download each version of the Image. As with the main Images Directory page, Production Images will have a message reading "Contact us for access to this image" with a link to Chainguard's [contact form](https://www.chainguard.dev/contact?utm_source=docs).
+* **Last changed**: when each version of the Image was last updated.
+
+Above the table is a search box which you can use to filter the different versions available for the Image. There is also a **Category** box you can use to filter out Production or Developer Images.
+
+### Overview Tab
+
+The **Overview** tab contains the Image's README. Typically, this will include instructions on how to download the Image, any relevant compatibility notes, and instructions on how to get started with using the Image.
+
+### Provenance Tab
+
+All Chainguard Images contain verifiable signatures and high-quality [software bills of materials](/open-source/sbom/what-is-an-sbom/) (SBOMs). These features allow you to confirm the origin of each Image and provide you with a detailed list of everything included in the Image.
```

Comment:
For awareness: There's an issue with using the verify instructions for organization images. https://github.com/chainguard-dev/console-ui/issues/2094

Reply:
Thanks for looping me in on this @garrying! This may be something DevEnablement can help with, but I need to ask around and get some more clarity on who owns the provenance template and how it gets sourced.

I'm going to resolve this conversation so this PR can move forward but I will follow up with you soon regarding this provenance tab issue